### PR TITLE
[Pages] Functions: Dashboard upload not supported

### PR DESCRIPTION
**Pull request by Skye-31**

### CODE REVIEW

File: content/pages/platform/functions/get-started.md [hunk 1]
```diff
@@ -59,8 +59,8 @@ Additionally, use other Cloudflare products such as [D1](/d1/) (serverless DB) a
 After you have set up your Function, deploy your Pages project. Deploy your project by:
 
 * Connecting your [Git provider](/pages/get-started/#connect-your-git-provider-to-pages).
-* Using [Direct Uploads](/pages/platform/direct-upload/) from the Cloudflare dashboard.
 * Using [Wrangler](/workers/wrangler/commands/#pages) from the command line.
+* Note: Using [Direct Uploads](/pages/platform/direct-upload/) from the Cloudflare dashboard is currently **not supported** with Functions.
```
Comment (WalshyDev) [inline content/pages/platform/functions/get-started.md]:
```suggestion

{{<Aside type="note">}}
Using [Direct Uploads](/pages/platform/direct-upload/) from the Cloudflare dashboard is currently **not supported** with Functions.
{{</Aside>}}
```

**Review by WalshyDev — APPROVED** [review]
Good from my end, needs PCX

File: content/pages/platform/functions/get-started.md [hunk 2]
```diff
@@ -59,9 +59,12 @@ Additionally, use other Cloudflare products such as [D1](/d1/) (serverless DB) a
 After you have set up your Function, deploy your Pages project. Deploy your project by:
 
 * Connecting your [Git provider](/pages/get-started/#connect-your-git-provider-to-pages).
-* Using [Direct Uploads](/pages/platform/direct-upload/) from the Cloudflare dashboard.
 * Using [Wrangler](/workers/wrangler/commands/#pages) from the command line.
 
+{{<Aside type="note">}}
```
Comment (deadlypants1973) [inline content/pages/platform/functions/get-started.md]:
```suggestion
{{<Aside type="warning">}}
```

File: content/pages/platform/functions/get-started.md [hunk 3]
```diff
@@ -59,9 +59,12 @@ Additionally, use other Cloudflare products such as [D1](/d1/) (serverless DB) a
...
+Using [Direct Uploads](/pages/platform/direct-upload/) from the Cloudflare dashboard is currently **not supported** with Functions.
```
Comment (deadlypants1973) [inline content/pages/platform/functions/get-started.md]:
```suggestion
[Direct Uploads](/pages/platform/direct-upload/) is currently not supported with Functions.
```

**Review by deadlypants1973 — APPROVED** [review]

**Pull request merged**